django_declarative_apis/machinery/__init__.py

```diff
@@ -213,13 +244,18 @@ def get_response(self):  # noqa: C901
                     immediate_task.run(self.bound_endpoint)
 
             except errors.ClientError as ce:
-                if ce.save_changes and resource and resource.is_dirty():
-                    resource.save()
+                if (
+                    ce.save_changes
+                    and resource
+                    and hasattr(resource, "is_dirty")
```

Comment:
This was a missing `hasattr` which allows DDA to work with models that don't use DirtyFields

Reply:
(unrelated to Django 3)